truss/cli/cli.py

```diff
@@ -471,17 +444,33 @@ def chains_init(
     user_friendly_path = os.path.join(target_directory, FILENAME)
     rich.print(f"Creating {user_friendly_path}...\n")
 
-    os.mkdir(abs_path)
+    source_code = _load_example_chainlet_code()
+
+    if not os.path.exists(abs_path):
+        os.mkdir(abs_path)
     with open(filename, "w") as f:
-        f.write(HELLO_PY)
+        f.write(source_code)
 
     rich.print(
         "Next steps:\n",
-        f"Run `python {user_friendly_path}` locally by adding `chains.run_local()` to your main.py file\n",
+        f"Run `python {user_friendly_path}` to execute the code locally\n",
         f"Run `truss chains deploy {user_friendly_path}` to deploy the chain to Baseten\n",
     )
 
 
+def _load_example_chainlet_code() -> str:
+    try:
+        from truss_chains import example_chainlet
+    # If `example_chainlet` fails validation and `_load_example_chainlet_code` is
+    # called as a result of that, we have a circular import ("partially initialized
+    # module 'truss_chains.example_chainlet' ...").
+    except Exception as e:
+        raise Exception("failed to load starter code. Please notify support") from e
```

Comment:
The comment here is not applicable anymore.
In `framework.py` the example code is loaded when as user-defined chainlet fails validation in order to show a correct example as part of the error message. But that requires that the example itself doesn't fail validation (that's where the circularity could ocur).

In the case here there would just be a validation error raised, if the example is faulty, but that would not recursively call `_load_example_chainlet_code`.

Nit: Capitalize messages and end with punctuation.

Reply:
sounds good - thanks for the heads up on that!

Reply:
updated the comment and punctuation